app/controllers/staff/organisations_controller.rb

```diff
@@ -17,10 +17,22 @@ def show
     project_activities = FindProjectActivities.new(organisation: organisation, current_user: current_user).call
     third_party_project_activities = FindThirdPartyProjectActivities.new(organisation: organisation, current_user: current_user).call
 
-    @fund_activities = fund_activities.map { |activity| ActivityPresenter.new(activity) }
-    @programme_activities = programme_activities.map { |activity| ActivityPresenter.new(activity) }
-    @project_activities = project_activities.map { |activity| ActivityPresenter.new(activity) }
-    @third_party_project_activities = third_party_project_activities.map { |activity| ActivityPresenter.new(activity) }
+    respond_to do |format|
+      format.html do
+        @fund_activities = fund_activities.map { |activity| ActivityPresenter.new(activity) }
+        @programme_activities = programme_activities.map { |activity| ActivityPresenter.new(activity) }
+        @project_activities = project_activities.map { |activity| ActivityPresenter.new(activity) }
+        @third_party_project_activities = third_party_project_activities.map { |activity| ActivityPresenter.new(activity) }
+      end
+      format.xml do
+        @project_activities = if third_party_project_activities.present?
```

Comment:
I wonder if we could make a more reliable check by looking at the `organisation.organisation_type` field which should include one of the values from this list: http://reference.iatistandard.org/203/codelists/OrganisationType/. I'd worry that in the future if we changed out minds and allowed the users to add more information, even though we chose not to publish it, we'd inadvertently wipe out all project information from IATI.

I'd be interested to know if we could say `if @organisation.is_non_government?`which checked the values against a whilist for a code of 21, 22, 23 or 24. I'd have to check with Dan if that assertion or similar could be made.

Reply:
At the moment we have `is_government?` which checks for codes 10 & 11. I'm *assuming*. that non-government is the opposite?

Reply:
After discussion with Odeta https://dxw.slack.com/archives/CN0H2L066/p1589798935095100 it looks like the govermental/governmental switch might be on whether or not we export *level B* activities, not level C/D

Reply:
After more discussion I'm going to add a "Download" button to Level C & Level D activities (with appropriate content around the buttons) so that the transparency user can choose the most appropriate level to download